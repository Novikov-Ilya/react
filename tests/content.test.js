import React from 'react';
import renderer from 'react-test-renderer';
import Content from '../src/components/content';

it('content component is rendered normally', () => {
    const component = renderer.create(
        <Content/>
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
})