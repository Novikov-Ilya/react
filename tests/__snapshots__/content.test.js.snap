// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`content component is rendered normally 1`] = `
<div
  className="content"
/>
`;
